src/ml/data/dwh/explore_dwh.py

```
@@ -114,41 +116,43 @@ def explore_dwh(database_path: str):
             print(f"総住宅数: {stats[0]:,}")
             print(f"平均価格: ${stats[1]:,.2f}")
             print(f"最低価格: ${stats[2]:,.2f}")
             print(f"最高価格: ${stats[3]:,.2f}")
             print(f"平均面積: {stats[4]:,.0f} sqft")
             print(f"平均単価: ${stats[5]:,.2f}/sqft")
-        
+
         # Location analysis
-        if 'v_location_analytics' in view_names:
+        if "v_location_analytics" in view_names:
             print(f"\n📍 地域別分析:")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)